gradio/blocks.py

```diff
@@ -2307,8 +2307,10 @@ def get_api_info(self):
                 # The config has the most specific API info (taking into account the parameters
                 # of the component), so we use that if it exists. Otherwise, we fallback to the
                 # Serializer's API info.
-                info = self.get_component(component["id"]).api_info()
-                example = self.get_component(component["id"]).example_inputs()
+                comp = self.get_component(component["id"])
```

Comment:
nit but let's avoid abbreviations like `comp` . Could rename component -> component_config, and then comp -> component